SmartSyncExplorerSwift/install.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+var packageJson = require('./package.json')
```

Comment:
Where is the package.json file?

Reply:
@wmathurin the package.json is checked in too.

Reply:
Indeed ;-)